baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggableExceptions.java

```diff
@@ -41,6 +41,7 @@
         linkType = BugPattern.LinkType.CUSTOM,
         severity = BugPattern.SeverityLevel.WARNING,
         summary = "Throw SafeLoggable exceptions to ensure the exception message will not be redacted")
+@SuppressWarnings("PreferSafeLoggableExceptions")
```

Comment:
this was undeterministically flaking compilation locally from gradle.. :(

Reply:
I think we should disable the check on the whole project

Reply:
it is already off, the baseline plugin itself turns it off for projects applying `java-gradle-plugin`.. I verified this with a snippet like so:
```
tasks.withType(JavaCompile) {
    println "${it.path}: ${options.errorprone.checks['PreferSafeLoggableExceptions']}"
}
```
and they are all OFF, however it doesn't seem to always take effect somehow